static/src/javascripts/projects/common/modules/experiments/tests/masthead-with-highlights.ts

```diff
@@ -0,0 +1,34 @@
+import type { ABTest } from '@guardian/ab-core';
+
+// Audience set to zero while setting up
+export const mastheadWithHighlights: ABTest = {
```

Comment:
Just double checking - these are client side ab-tests but if we want to affect the server side render for the header, nav etc I think they would need to be server side tests (aka `Experiments.scala`).